assignments/array-methods.js

```diff
@@ -56,28 +56,60 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(currentValue) {
+    fullName.push(`${currentValue.first_name}  ${currentValue.last_name}`);
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(function(currentValue) {
```

Comment:
Array methods look good. If you want, you can personalize the parameters for the function you are working on. 
Ex: runners.forEach(name => fullName.push(name.first_name + '  ' + name.last_name ));
runners.sort()